tools/abel/README.md

```diff
@@ -0,0 +1,146 @@
+# ABEL
+### Automated Building Entity Loader
+
+Allows system integrators to easily and efficiently create and
+modify [Building Configuration files](../../ontology/docs/building_config.md).
+
+## Setup and installation
+
+The setup process is broken into two parts:
+
+First, setup a virtual environment using [virtualenv](https://virtualenv.pypa.io/en/latest/) and install all of the
+proper dependencies. Then obtain credentials for [Google Sheets API](https://developers.google.com/sheets/api/reference/rest) for use with
+ABEL. Total setup process should only take about 15 minutes.
+
+### Set up a tooling environment
+
+1. Clone the Digital Buildings repository
+
+  ```
+  git clone https://github.com/google/digitalbuildings.git
+  ```
+
+2. Navigate to the [digitalbuildings tooling library](../../tools/)
+
+  ```
+  cd digitalbuildings/tools/
+  ```
+
+3. Create a virtual environment with an environment name, in this example: `tooling`
+
+  ```
+  virtualenv tooling
+  ```
+
+4. Activate your virtual environment
+
+* MacOs/Linux:
+
+  ```
+  source tooling/bin/activate
+  ```
+
+* Windows:
+
+  ```
+  tooling/Scripts/activate
+  ```
+
+5. Depending on your operating system, run either of the global setup scripts to configure dependencies
+
+* Linux/MacOs:
+  ```
+  bash pip_install.sh
+  ```
+
+* Windows:
+
+  ```
+  pip_install.bat
+  ```
+
+### Obtain a spreadsheet token for your google sheets account
+
+1. Contact your IoT Technical Program Manager and ask to be added to a GCP service account with `Service Account User` and `Service Account Token Creator` roles
```

Comment:
this should not be put here it is leaking how we do stuff.
This will be automated, the iot tpm will not need to know the details.

Reply:
Removed the roles.